zerver/views/realm.py

```diff
@@ -2,7 +2,6 @@
 
 from typing import Any, Optional
 
-from django.http import HttpRequest, HttpResponse
```

Comment:
(this is probably part of the cause of this failing the `static analysis` build in Travis CI